tftrt/benchmarking-cpp/main.cc

```diff
@@ -167,6 +179,7 @@ int main(int argc, char* argv[]) {
   string model_path = "/path/to/model/";
   string signature_key = "serving_default";
   int32_t batch_size = 64;
+  int32_t input_size = 128;
```

Comment:
Hmm @nvkevihu  -- is it expected that all inputs of unknown shape will be of the same size?

Reply:
For the object detection and transformer models, yes (the obj detection models generally take square input images; e.g. 640x640, and the transformer models just need to specify sequence length). There wasn't a need to handle other shapes, so I chose to go with the simplest solution.

Reply:
sounds good.